data/FilterList.json

```diff
@@ -2325,7 +2321,6 @@
     "updatedDate": "2010-05-18T00:00:00",
     "viewUrl": "http://www.squirrelconspiracy.net/abp/facebook-privacy-list.txt",
     "viewUrlMirror1": "https://raw.githubusercontent.com/DandelionSprout/adfilt/master/AncientLibrary/Facebook%20Privacy%20List.txt",
```

Comment:
```suggestion
    "viewUrlMirror1": "https://raw.githubusercontent.com/DandelionSprout/adfilt/master/AncientLibrary/Facebook%20Privacy%20List.txt"
```

Reply:
This suggestion helped me notice a pattern that I've had to fix after I did some automated text removal in the initial PR. *This* time it should be fixed.